ra/ra_test.go

```diff
@@ -2011,8 +2056,10 @@ func TestRecheckCAAEmpty(t *testing.T) {
 func TestRecheckCAASuccess(t *testing.T) {
 	_, _, ra, _, cleanUp := initAuthorities(t)
 	defer cleanUp()
-	names := []string{"a.com", "b.com", "c.com"}
-	err := ra.recheckCAA(context.Background(), names)
+	rechecks := []string{
```

Comment:
There are a few places where you renamed `names` to `rechecks` and slightly reformatted to accompany the `caaRecheck` struct. Now those changes are reverted, but some of the name changes and added newlines stay the same. This is pretty nitpicky, but could you revert these too? I think it will help see the "meaningful" changes in this PR, and keep the `git blame` clean. Another example is in `ra.recheckCAA`.

Reply:
n/p! Fixed in fe79c02